thirdparty/vars-x86_64.sh

```diff
@@ -46,10 +46,10 @@ ASYNC_PROFILER_SOURCE="async-profiler-2.9-linux-x64"
 ASYNC_PROFILER_MD5SUM="29127cee36b7acf069d31603b4558361"
 
 # jindosdk for Aliyun OSS
-JINDOSDK_DOWNLOAD="https://cdn-thirdparty.starrocks.com/jindosdk-4.6.8-linux.tar.gz"
-JINDOSDK_NAME="jindosdk-4.6.8-linux.tar.gz"
-JINDOSDK_SOURCE="jindosdk-4.6.8-linux"
-JINDOSDK_MD5SUM="5436e4fe39c4dfdc942e41821f1dd8a9"
+JINDOSDK_DOWNLOAD="https://jindodata-binary.oss-cn-shanghai.aliyuncs.com/release/6.5.0/jindosdk-6.5.0-linux.tar.gz"
```

Comment:
using this: https://cdn-thirdparty.starrocks.com/jindo/jindosdk-6.5.0-linux.tar.gz